src/basic_recipes/pie.jl

```diff
@@ -37,25 +50,26 @@ function plot!(plot::Pie)
         end
 
         # create vector of a vector of points for each piece
-        vertex_arrays = map(boundaries[1:end-1], boundaries[2:end]) do sta, en
+        vertex_arrays = map(boundaries[1:end-1], boundaries[2:end], xs, ys, offset_radius) do sta, en, x, y, r
+            x += cos((en + sta) / 2 + offset) * r
+            y += sin((en + sta) / 2 + offset) * r
             distance = en - sta
             # how many vertices are needed for the curve?
             nvertices = max(2, ceil(Int, rad2deg(distance) * vertex_per_deg))
 
             # curve points
             points = map(LinRange(sta, en, nvertices)) do rad
-                Point2f(cos(rad + offset), sin(rad + offset)) .* radius
+                Point2f(cos(rad + offset) * radius + x, sin(rad + offset) * radius + y)
```

Comment:
This isn't related to the pr but it would be good to also replace the `Point2f` calls with `Point2` for Float64 compatability.